drivers/can/can_mcp2515.c

```diff
@@ -352,6 +371,25 @@ static int mcp2515_configure(struct device *dev, enum can_mode mode,
 		goto done;
 	}
 
+	/* Wait 128 OSC1 clock cycles at 1MHz (minimum clock in frequency)
+	* see MCP2515 datasheet section 8.1 Oscillator Start-up Timer
+	*/
```

Comment:
Slight nit here, Zephyr follows the Linux kernel commenting style: https://www.kernel.org/doc/html/latest/process/coding-style.html#commenting
The asterisk characters should be aligned.